### PR TITLE
SAVAMC2-87 SAVAMC2-120 Update Patient Tables

### PR DESCRIPTION
**Pull request by ChristineDuong**
# Description
Jira Ticket: SAVAMC2-87 SAVAMC2-120

SAVAMC2-87: On the "All Monitorees" tab, you can now sort by status. This is sorted alphabetically at the moment.
SAVAMC2-120: Public health dashboard now shows updated vaccine headers. End of monitoring updated to be 1 year out of Dose 2. 

Additionally:
* Fixed a bug on editting vaccine dosages. Before the validate function would crash when trying to change values in the edit view due to type issues.

# Testing

NOTE: This assumes that a dosage 2 is administered after a dose 1 and that if you acquire a dose 2, you will not be acquiring another dose 1. This restriction isn't in place yet, because I'm assuming it'll be covered in SAVAMC2-16.

This fix was tested on the following browsers (submitter must check all those that apply):
* [x] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] IE11

Please describe the tests needed to verify your changes. Provide instructions for reproducing these tests. List any relevant details for your test configuration.

### CODE REVIEW

File: app/javascript/components/dosage/Dosage.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class Dosage extends React.Component {
   }
 
   validate = () => {
+    console.log(this.state);
```
Comment (jkufro) [inline app/javascript/components/dosage/Dosage.js]: Can this be removed?

File: app/javascript/components/dosage/Dosage.js [hunk 2]
```diff
@@ -74,12 +75,14 @@ class Dosage extends React.Component {
...
     ];
-    let check = Object.keys(this.state).filter(k => fields.includes(k) && (this.state[`${k}`] == null || this.state[`${k}`].trim() == ''));
+
+    let check = Object.keys(this.state).filter(k => fields.includes(k) && (this.state[`${k}`] == null || String(this.state[`${k}`]).trim() == ''));
     this.setState({ dosageInvalid: check.length > 0 });
   };
 
   handleChange(event) {
     this.setState({ [event.target.id]: event.target.value }, () => {
+      console.log(this.state);
```
Comment (jkufro) [inline app/javascript/components/dosage/Dosage.js]: Can this be removed?

**Review by jkufro — CHANGES_REQUESTED** [review]
verified the changes and they look good. Just a couple of stray `console.log` lines that could be removed. I'm fine with the commented out lines in case we end up deciding we need them again later.

**Review by jkufro — APPROVED** [review]

File: app/javascript/components/dosage/Dosage.js (hunk 2 above)
Comment (ChristineDuong) [reply app/javascript/components/dosage/Dosage.js]: Removed in bc87dee

**Pull request merged**